code/CLI/CLIEditor.py

```diff
@@ -97,22 +97,182 @@ def do_list_classes(self, args):
 
     def do_create_field(self, args):
         executeCMD(self.model, self.command_history, "create_field", args.split())
+
+    def complete_create_field(self, text, line, begidx, endidx):
+        classes = fetch_classes(self.model)
+        # split the command into a list of words
+        args = line.split()
+        # if command has 1 word
+        if len(args) == 1:
+            return classes
+        # if command has 2 words
+        elif len(args) == 2:
+            # if second word is not a part of the class list, complete it into one if possible
+            if args[1] not in classes:
+                return [
+                        model_class for model_class in classes
+                        if model_class.startswith(text)
+                    ]
+            # if second word is a part of the class list, show a list of visibilities
+            else:
```

Comment:
If you implement the argIndex that I suggested, then this else can be removed and the above `if args[1] not in classes` can also be removed